scripts/staticData.js

```diff
@@ -2,25 +2,37 @@
+    image: '../assets/images/testimonial1.png',
+    task: 'Desk Moving',
+    comment: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  },
+  {
+    name: 'Elias Kindeya',
+    image: '../assets/images/testimonial1.png',
+    task: 'Desk Moving',
+    comment: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  },
+  {
+    name: 'Marta Mehari',
+    image: '../assets/images/testimonial2.png',
```

Comment:
- [ ] I am sorry that page isn't working well at this time but I am sure you get this fixed in a short while. Good job attempting to apply previous recommendations but it                                                                                                           seems to need a little more attention as the images of speakers are not showing :confused: 

![image](https://github.com/habitoz/serategna/assets/99715640/29ce84c6-78e3-424f-b49e-312978919d15)

- [ ] Also, like the latest reviewer mentioned, your `about.html` is working now but the design is still not according to the requirement. Kindly check this [video](https://watch.screencastify.com/v/UXCanjch9VzsmJE3wDqp) and apply the fix. Like you will see, it appears that the attribute of your `href` for home isn't working. Just like you fixed the `about.html` which the reviewer pointed out :wink: You gat this!
- [ ] As mentioned by previous code reviewer, your homepage seems to be too different from the design, kindly fix this.
- [ ] [OPTIONAL] I am suggesting that you use real but imaginary content for your page instead of some random lorem texts... Just imagine if a recruiter is interested in you and asks about your recent project... :thinking: